src/OpenLoco/src/Drawing/DrawingContext.h

```diff
@@ -148,6 +148,8 @@ namespace OpenLoco::Drawing
 
         virtual void drawStringYOffsets(Gfx::RenderTarget& rt, const Ui::Point& loc, AdvancedColour colour, const void* args, const int8_t* yOffsets) = 0;
 
+        virtual void drawStringTicker(Gfx::RenderTarget& rt, const Ui::Point& origin, string_id stringId, Colour colour, uint8_t numLinesToDisplay, uint16_t numCharactersToDisplay, uint16_t width) = 0;
```

Comment:
I'm honestly not a fan of having the drawing context have such a specific function, can't we have this code outside the context?

Reply:
it would be a little hard to rework it that way

Reply:
What would be the pain point? It doesn't look like its doing anything special in there, passing the drawing context as the argument should keep this function almost the same.